main.go

```diff
@@ -0,0 +1,94 @@
+/**
+ *Author: Stefan
+ *Date: 11/24/2019
+ *Last changes: 11/26/2019 10.23
+ *Task: Implement sort.Interface interface for People type.
+ *				Sort people in order of increasing age.
+ *				If two people have the same age - sort them by name.
+ *				Add test cases.
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+//Person struct
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+}
+
+//People struct
+type People []Person
+
+//Len method
+func (p People) Len() int {
+	return len(p)
+}
+
+//Less method
+func (p People) Less(i, j int) bool {
+
+	if p[i].birthDay.Sub(p[j].birthDay) > 0 {
+		return true
+	} else if p[i].birthDay.Sub(p[j].birthDay) < 0 {
+		return false
+	} else if p[i].firstName < p[j].firstName {
+		return true
+	} else if p[i].firstName > p[j].firstName {
+		return false
+	}
+	return p[i].lastName < p[j].lastName
+}
+
+//Swap method
+func (p People) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+}
+
+//Interface interface which implement some methods from gplang.org/pkg
+type Interface interface {
```

Comment:
It's better to use more meaningful name, like `Sorter`

Reply:
This interface is already defined within the `sort` package. Why we need to redefine it here?

Reply:
> This interface is already defined within the `sort` package. Why we need to redefine it here?
YES. Done!)